### ISSUE TITLE
Enhance AMP debug responses

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#2406]
AMP requests don’t currently have a way to report rejects that happen within the modularity workflow.

When &debug=1 is on the /amp endpoint, right now only `ext.debug` is supplied on the response. I propose we send the whole seatbid array as well – any ORTB that would have been sent to the /auction endpoint should be there in AMP debug mode.

e.g.
```
{
    "targeting": {
         ...
    },
    "ext": {
        ... entire PBS ORTB response here ...
    }
}
```
If there are automated tests hitting the debug endpoint and parsing the results, then this would be a breaking change because ext.debug would now be ext.ext.debug.

I think we could just announce this change because it's not going to change production behavior, but there are some things we could do to ease into it:
- only do this new behavior when `&verbose=1` is in the header
- move ext.ext.debug up to ext.debug

**Comment by bretg** [comment]
Discussed in committee. Resolved to move to `ortb2` instead of `ext` and **not** implement the &verbose=1 flag. 

Anyone who's built a monitor using the debug field, will need to update the location where they're looking for debug.

**Comment by bsardo** [comment]
@Net-burst we discussed this in backlog. The group doesn't think we should tackle this one but since it has the 4.0 label on it I wanted to check with you. Thoughts?

**Comment by Net-burst** [comment]
I agree with that, @bsardo. IDK whether we should close the issue or not. But I have my doubts that this will ever be needed.
